src/tools/rustfmt/src/bin/main.rs

```diff
@@ -948,6 +944,8 @@ mod test {
         let config_file = Some(Path::new("tests/config/style-edition/overrides"));
         let config = get_config(config_file, Some(options));
         assert_eq!(config.style_edition(), StyleEdition::Edition2024);
+        // FIXME: this test doesn't really exercise anything, since
+        // `overflow_delimited_expr` is disabled by default in edition 2024.
         assert_eq!(config.overflow_delimited_expr(), false);
```

Comment:
Might make sense to remove this test outright and the `style_edition_defaults_overridden_from_cli` test below, but  I'm fine just adding the FIXME.

Reply:
Let's leave it